ansible/golden_config_db/dhcp_server_mx.json

```diff
@@ -0,0 +1,243 @@
+{
+    "DHCP_SERVER_IPV4": {
+        "Vlan1000": {
+            "gateway": "192.168.0.1",
+            "lease_time": "900",
+            "mode": "PORT",
+            "netmask": "255.255.255.0",
+            "state": "enabled"
+        }
+    },
+    "DHCP_SERVER_IPV4_PORT": {
+        "Vlan1000|Ethernet0": {
```

Comment:
Are the port config the same across all m0 and mx platform?

Reply:
No, hence updated port_alias to get index_portname_map. Generate this file by map